core/src/test/java/io/jenkins/pluginhealth/scoring/probes/ThirdPartyRepositoryDetectionProbeTest.java

```diff
@@ -26,19 +25,31 @@ ThirdPartyRepositoryDetectionProbe getSpy() {
         return spy(ThirdPartyRepositoryDetectionProbe.class);
     }
 
-    static Stream<Arguments> pomPathAndProbeResults() {
+    static Stream<Arguments> pomPathAndProbeSuccessTestParameters() {
+        return Stream.of(
```

Comment:
I tried to parameterize the success test case. But `shouldPassIfNoRepositoriesDetected` would fail.